src/libsyntax/errors/emitter.rs

```diff
@@ -663,7 +663,7 @@ fn stderr_isatty() -> bool {
     type DWORD = u32;
     type BOOL = i32;
     type HANDLE = *mut u8;
-    const STD_ERROR_HANDLE: DWORD = -12i32 as DWORD;
+    const STD_ERROR_HANDLE: DWORD = -12 as DWORD;
```

Comment:
Yep, `DWORD` is `u32` so the type has to be specified as `i32` in order to specify a negative value and then cast to `DWORD`.